release/models/platform/openconfig-platform-pipeline-counters.yang

```diff
@@ -1093,6 +1099,14 @@ module openconfig-platform-pipeline-counters {
         packets are dropped due to legitimate forwarding decisions (ACL drops,
         No Route etc.)";
     }
+
+    leaf urpf-aggregate {
+      type oc-yang:counter64;
+      description
+        "This aggregation of counters represents the conditions in which
+        packets are dropped due to failing uRPF lookup check.";
```

Comment:
Openconfig operators feedback  is to clarify the expected behavior of the other aggregate counters.

```suggestion
        "This aggregation of counters represents the conditions in which
        packets are dropped due to failing uRPF lookup check.  This counter
        and the packet-processing-aggregate counter should be incremented
        for each uRPF packet drop.";
```